packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js

```diff
@@ -281,6 +294,7 @@ PieWidgetUI.propTypes = {
   colors: PropTypes.array,
   formatter: PropTypes.func,
   tooltipFormatter: PropTypes.func,
+  percentFormatter: PropTypes.func,
```

Comment:
Export the new prop from types `packages/react-ui/src/types.d.ts`

Reply:
Done!